File: iOS - Using Objective-C/day3/day3Task1_CombinedTabbedAndNavigaionBr/day3Task1_CombinedTabbedAndNavigaionBr/SecondViewController.m
//
//  SecondViewController.m
//  day3Task1_CombinedTabbedAndNavigaionBr
//
//  Created by Uef on 19/12/2024.
//

#import "SecondViewController.h"
#import "ThirdViewController.h"
@interface SecondViewController ()

@end

@implementation SecondViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)GgoToThird:(id)sender {
    ThirdViewController * tVC = [self.storyboard instantiateViewControllerWithIdentifier:@"third"];
    [self.navigationController pushViewController:tVC animated:YES];
    
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: Swift/Day2/day2_task2/day2_task2/SecondViewController.swift
//
//  SecondViewController.swift
//  day2_task2
//
//  Created by Uef on 08/01/2025.
//

import UIKit

class SecondViewController: UIViewController {
    @IBOutlet weak var movTitleLB: UILabel!
    @IBOutlet weak var movImageView: UIImageView!
    @IBOutlet weak var movRatingLB: UILabel!
    @IBOutlet weak var releaseYearLB: UILabel!
    @IBOutlet weak var genreLB: UILabel!
    var movie : Movie?
    override func viewDidLoad() {
        super.viewDidLoad()
        if let movie = movie {
            movTitleLB.text = movie.title
            movImageView.image = UIImage(named: movie.image)
            movRatingLB.text = String(movie.rating)
            releaseYearLB.text = String(movie.realeaseYear)
            if !movie.genre.isEmpty {
                            genreLB.text = movie.genre.joined(separator: ", ")
                        }
        }
        // Do any additional setup after loading the view.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
